dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -265,43 +265,66 @@ public void exportToORCID(@Parameter(hidden = true, name = "user") @Auth User us
         String orcidWorkString;
         try {
             orcidWorkString = ORCIDHelper.getOrcidWorkString(entry, optionalVersion, putCode);
-        } catch (JAXBException | DatatypeConfigurationException e) {
-            throw new CustomWebApplicationException("Could not export to ORCID: " + e.getMessage(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
-        }
-        try {
             if (putCode == null) {
                 createOrcidWork(optionalVersion, entry, user, orcidWorkString, orcidByUserId);
             } else {
-                updateOrcidWork(user, orcidWorkString, orcidByUserId, putCode);
+                boolean success = updateOrcidWork(user, orcidWorkString, orcidByUserId, putCode);
+                if (!success) {
+                    LOG.info("Could not find ORCID work based on put code");
```

Comment:
Is this going to be common? If not, I suggest warn or error for the log message.